modin/experimental/xgboost/xgboost.py

```diff
@@ -82,22 +94,25 @@ def predict(
         num_actors: Optional[int] = None,
         **kwargs,
     ):
-        """
-        Run prediction with a trained booster.
+        """Run distributed prediction with a trained booster.
+
+        During work it evenly distributes `data` between workers,
+        runs xgb.predict on each worker for subset of `data` and creates
+        Modin DataFrame with prediction results.
 
         Parameters
         ----------
-        data : DMatrix
+        data : modin.experimental.DMatrix
```

Comment:
```suggestion
        data : modin.experimental.xgboost.DMatrix
```
I think this is more correct, is it not?

Reply:
fixed